modeler.js

```diff
@@ -0,0 +1,131 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.MODELER_CLIENT_ID,
+  clientSecret: process.env.MODELER_CLIENT_SECRET,
+  audience: process.env.MODELER_AUDIENCE
+};
+
+// An action that creates a new project.
+async function createProject([projectName]) {
```

Comment:
non-blocking conversation: 

One thing I learned while testing this, and it is documented in [the Modeler API's swagger UI](https://modeler.cloud.camunda.io/swagger-ui/index.html#/Projects/createProject) -- when creating a new project through the API, it has no collaborators, and therefore can't be viewed through the Modeler UI until you add collaborators via a different API endpoint. I discovered this, because I couldn't find my project in the UI after creating it 😅

I'm not sure what that means for this function. Do you think it's fine that the user won't be able to find their project in the UI after creating it? _I_ tried to find mine; I imagine others might do the same. Should we add an argument to this function for the collaborator and make a call to the second endpoint for them after creating the project? Are projects maybe not the panacea that I thought they were, and we should look for _another_ resource that can be created with no extra effort?

Reply:
> Do you think it's fine that the user won't be able to find their project in the UI after creating it? 

No, I agree here. Especially from a beginner's perspective, this could lead to a lot of confusion on their end.

> Should we add an argument to this function for the collaborator and make a call to the second endpoint for them after creating the project?

The simple fix here is to do this, and likely add an admonition explaining why the script looks like this. Does it solve the issue? Yes. Does it provide a clean tutorial? Not really, but it would mean we could merge this, and add an issue to the backlog to clean up. A "meh" option IMO.

> Are projects maybe not the panacea that I thought they were, and we should look for another resource that can be created with no extra effort?

Would it make sense to have the users create a project, then add a collaborator (PUT), then search for a collaborator (POST)?

I know @pepopowitz is out for a few weeks, so I'll loop in @akeller. This isn't urgent, so if you have time to take a look and provide feedback, I might be able to move forward with this and merge in a solution before the August alpha 👍

Reply:
This is a point of friction, and we should encourage users to create a project and immediately add a collaborator so they can find it in the UI. I would also like you to take this feedback to the Modeler API team. I do not see an obvious reason why you would want to create a project with no collaborators, so could we _elect_ one _automagically_? That would be a more intuitive experience IMO. 

I would chain these exactly as you mentioned, @christinaausley. And absolutely be explicit that if they don't add a collaborator, they will not see this in the UI. The earlier they build that habit, the less likely they are to fall into that trap.

Reply:
Made these changes -- will add a note on the collaborator discussion when this is approved and I can clean up the docs PR 👍